index.html

```diff
@@ -28,10 +27,9 @@
         <div class= "content">
 
```

Comment:
- [ ] Please use a `section` tag to wrap inside the title, the about me, and the links. This will help you to organize the HTML and the page properly. 👍 💪 
Remember that you must use one of the Figma templates.